djstripe/apps.py

```diff
@@ -19,8 +19,6 @@ class DjstripeAppConfig(AppConfig):
     def ready(self):
         import stripe
 
-        from . import checks, event_handlers
```

Comment:
These imports are critical to register checks and event handlers. Needs a noqa and a comment.